packages/api/README.md

```diff
@@ -17,17 +17,31 @@ If you want to run the API independently and locally, the following technologies
 
 ## Steps to start the API
 
-1. fork or clone the repository locally
-2. assuming you are in the folder containing your clone, go to the terminal and run
+Complete the following steps **after provisioning the Azure resources** in order to develop and debug against Azure resources.
 
-```bash
-cd packages/api && npm install
-```
-Now you have all the dependencies installed for the API and can run
+1. Fork or clone the repository locally
+2. Provision Azure resources. 
 
-```bash
-npm start
-```
```

Comment:
these deleted steps are targeting the local dev setup (using the provided devcontainer setup). I don't think we should delete them. 

Developing against provisioned Azure PG instance could be added in a different section. WDYT?

Reply:
Reverted